client/apollo/react/src/Form/InputTextAtom/InputTextAtomCommon.tsx

```diff
@@ -32,17 +32,17 @@ const InputTextAtom = forwardRef<HTMLInputElement, InputTextAtomProps>(
     },
     inputRef,
   ) => {
-    const componentClassName = getComponentClassName(
-      "af-form__input-text",
+    const componentClassName = getClassName({
+      baseClassName: "af-form__input-text",
+      modifiers: [classModifier + (error || ariaErrormessage ? "error" : "")],

```

Comment:
c'est un tableau, chaque modifier est un élément du tableau :

```suggestion
      modifiers: [classModifier, (error || ariaErrormessage ? "error" : "")],
```